src/controllers/userController.js

```diff
@@ -85,13 +109,37 @@ const updateUserPassword = async (req, res) => {
   await user.save(); // Save the updated user to the database
   // Send a JSON response with the status code 200 OK and a success message
   res.status(StatusCodes.OK).json({ msg: 'Success! Password Updated.' });
-};
 
+  /*
+     #swagger.summary = 'Fetch the current requesting user by id, update their password and save it to the database.'
+     #swagger.responses[200] = {
+				description: 'User successfully fetched and password updated.',
+				type: 'object'
+				properties: {
+				  msg: { type: 'string' }
+				}
```

Comment:
my earlier advice here was incorrect; this should just be a `schema` object with an example of the expected response (which swagger-autogen will analyze to determine the types of the fields):

```suggestion
				schema: { msg: 'Success! Password Updated.' }
```